packages/ai/src/ui/ui-messages.ts

```diff
@@ -179,21 +179,27 @@ export type ToolUIPart<TOOLS extends UITools = UITools> = ValueOf<{
         state: 'input-streaming';
         input: DeepPartial<TOOLS[NAME]['input']>;
         providerExecuted?: boolean;
+        output: undefined;
+        errorText: undefined;
```

Comment:
`undefined` or `never`? (i have used `never` in other places for something similar)

Reply:
and you might want it to be optional

Reply:
I use never now, tested the destructing and it still works